### PR TITLE
Sb 43 dark mode

### PR DESCRIPTION
**Pull request by raiot**
# What's the problem?
Dark mode was not enabled in all the application.

# What's the solution?
- Enable dark mode in all application
- Created a small localStorage function to store user preferences so they're kept after reloading the page
- Refactored some parts of the code

### CODE REVIEW

**Comment by imariel2d** [comment]
It looks pretty cool. 😄 

But I found some details 👀 

Dark_Theme


Dark_Theme_2


Dark_Theme_3

**Comment by raiot** [comment]
@ImAriel2D Cool cool, thanks for the info, I think I addressed all the issues in the last commit

**Review by imariel2d — APPROVED** [review]
LGTM 😄 

(The only problem is the failing task 😢 )

**Comment by tonymtz** [comment]
I think the failing check can be fixed by wrapping the component in the `StyleProvider`.

File: src/domains/audio-player/audio-progress/audio-progress.test.js [hunk 1]
```diff
@@ -1,26 +1,21 @@
 import React from 'react';
 import { Provider } from 'react-redux';
-import { ThemeProvider } from 'styled-components';
 import { render } from '@testing-library/react';
 
 import store from 'store';
 
 import AudioProgress from 'domains/audio-player/audio-progress/index';
+import StyleDecorator from '../../../stories/decorators/style-decorator';
```
Comment (tonymtz) [inline src/domains/audio-player/audio-progress/audio-progress.test.js]: this import can be absolute:

```jsx
import StyleDecorator from 'stories/decorators/style-decorator';
```

**Review by tonymtz — APPROVED** [review]
# **LGTM!!!**

**Pull request merged**